release/release.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+fetch_push_prior_release () {
+    git fetch $upstream_name --tags
+    LATEST_TAG=$(git tag --sort=-v:refname | head -n 1)
+    MODIFIED_TAG=$(echo "$LATEST_TAG" | awk -F. '{print $1 "." $2 ".x"}') # convert to [major].[minor].x format from [major].[minor].[patch]
+    git branch release/$MODIFIED_TAG $LATEST_TAG
+    git push $upstream_name release/$MODIFIED_TAG
+    git branch -D release/$MODIFIED_TAG
+}
+
+# append rc for release-candidate if necessary
+tag_and_push () {
+    final_version="v$VERSION"
+    if [ "$1" == "rc" ]; then
+        final_version+="-rc"
+    fi
+    git tag $final_version
+    git push $upstream_name $final_version
+}
+
+TYPES=(
+    "major"
+    "minor"
+    "patch"
+)
+
+UPSTREAM="https://github.com/devfile/registry-support.git"
```

Comment:
Does it make sense here to use an env var or something (with default this value) so we can be able to test it? E.g on our forks

Reply:
I just commented out the `UPSTREAM` var and replaced it with `UPSTREAM=<my github>` to test it, I initially strayed from keeping this as an env var because in its final use it'l never need to be set via env var and it reduces the variables to keep track of for the user. wdyt

Reply:
Yeah after a second thought let's keep it like this. After all is only for testing